geotools/src/main/scala/geotrellis/geotools/SimpleFeatureToFeature.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.geotools
+
+import geotrellis.vector._
+
+import com.vividsolutions.jts.{geom => jts}
+import org.opengis.feature.simple.SimpleFeature
+
+import scala.collection._
+import scala.collection.JavaConverters._
+import scala.reflect.ClassTag
+
+
+object SimpleFeatureToFeature {
+
+  private def jtsToGeotrellis[G <: Geometry : ClassTag](geometry: Object): Option[G] = {
+    geometry match {
+      case g: jts.Geometry => Some(Geometry(g).asInstanceOf[G])
+      case g => throw new Exception(s"Input $g is not a jts.Geometry")
+    }
+  }
+
+  def apply[G <: Geometry : ClassTag](simpleFeature: SimpleFeature): Feature[G, immutable.Map[String, Object]] = {
```

Comment:
I wonder about our ability to pull off a roundtrip without an inverse of the `transmute` function defined for the `Feature` -> `SimpleFeature` translation